loader.qml

```diff
@@ -1,57 +1,193 @@
 import QtQuick 2.15
 import QtGraphicalEffects 1.12
 import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
 
 ApplicationWindow {
     id: root
+    visible: true
+    minimumWidth: 640
+    minimumHeight: mysize + header.height // + menuBar.height
     property bool displayAmbient: false
     property bool round: true
+    property int mysize: 640
     property var testface: Qt.application.arguments[1]
+    readonly property var mouseWheelScale: 1/15
 
-    toolBar: ToolBar {
-        RowLayout {
-            anchors.fill: parent
-            anchors.margins: spacing
-            CheckBox {
-                id: roundCheckBox
-                text: "Round"
-                Component.onCompleted: checked = round
-                onCheckedChanged: round = checked; 
-            }
-            CheckBox {
-                id: checkBox12h
-                text: "12h time"
-                Component.onCompleted: checked = use12H.value
-                onCheckedChanged: use12H.value = checked; 
-            }
-            CheckBox {
-                id: ambientCheckBox
-                text: "Dark"
-                Component.onCompleted: checked = displayAmbient
-                onCheckedChanged: displayAmbient = checked; 
+        /*
+    menuBar: MenuBar {
```

Comment:
As a general rule, commented code is typically a bad idea (unless if it documents a function usage or something, and still... it's debatable whether that is even a good practice)
Either it's existing code that you want to delete but don't really dare to, in which case you should really just delete it because it will be in the git history anyway, so it's not really really gone.
Or it's useless code, in which case it should also just be deleted.
Same comment for line 11